### PR TITLE
Update: Note has been added to the document

### PR DESCRIPTION
**Pull request by kannanb-github**
The basic purchaser role in Microsoft Store for business doesn't have the ability to acquire apps. So the following note has been added to the document.
Currently, the Basic purchaser role is only available for schools using Microsoft Store for Education. For more information, see [Microsoft Store for Education permissions..](https://docs.microsoft.com/en-us/education/windows/education-scenarios-store-for-business?toc=/microsoft-store/education/).

### CODE REVIEW

**Comment by kannanb-github** [comment]
This PR closes for the issue https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3641

**Comment by martyav** [comment]
Hi, @kannanb-github 

I'm a writer with MS, and contacted the product team to verify if the Basic purchaser role is exclusive to Store for Education.

They claim it's also now available in Store for Business.

**Comment by kannanb-github** [comment]
Hi @martyav,

The basic purchaser role does have for Microsoft Store for business, but we cannot use it for acquiring apps. The basic purchaser role is still restricted as mentioned by the issue reporter of issue #3641. I tried this issue personally and find the same issue as the issue reporter escalated. So I added the note to add to the document with this PR.

Thanks,
Kannan

File: store-for-business/roles-and-permissions-microsoft-store-for-business.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ This table lists the roles and their permissions.
 | ------------| ------ | --------  | ------ |
 | Purchaser   | X      |   X       |      |
 | Basic purchaser |  X     |          | X      |
+>[!Note]
+    >Currently, the Basic purchaser role is only available for schools using Microsoft Store for Education. For more information, see [Microsoft Store for Education permissions..](https://docs.microsoft.com/en-us/education/windows/education-scenarios-store-for-business?toc=/microsoft-store/education/).
```
Comment (illfated) [inline store-for-business/roles-and-permissions-microsoft-store-for-business.md:77]: These lines need more mindful formatting if they are going to be added to this document:
```suggestion

> [!NOTE]
> Currently, the Basic purchaser role is only available for schools using Microsoft Store for Education. For more information, see [Microsoft Store for Education permissions](https://docs.microsoft.com/education/windows/education-scenarios-store-for-business).
```
Also note the simplification of the URL to keep the pages language agnostic, as well as removing redundant query at the end.

**Comment by martyav** [comment]
@kannanb-github I've relayed the information you gave me, and asked the team if there are ways a user might still be blocked from buying apps even if they're a Basic Purchaser. There definitely seems to be a lot of confusion around this.

**Pull request closed without merging**